core/utils/poly-look/src/visualisations/charts/chart.js

```diff
@@ -72,15 +72,15 @@ export class Chart {
       .reduce((prev, curr) => prev.toLowerCase() + "-" + curr.toLowerCase());
   }
 
-  createSVG() {
+  _createSVG() {
     return d3
       .select(this._selector)
       .append("svg")
       .attr("viewBox", `0 0 ${this.width} ${this.height}`);
   }
 
-  generateChart() {
-    return this.createSVG()
+  _generateChart() {
```

Comment:
Why did you make it "private"?

Reply:
Yeah I went with this convention everywhere else so as i was overwriting createSVG I figured that it was wrong here :D